config/jenkins.yaml

```
@@ -22,6 +22,12 @@ jenkins:
         env:
           - key: "ARENDER_DOCKER_REGISTRY"
             value: "docker-arender.packages.nuxeo.com"
+          - key: "ARENDER_HOST_DOCKER_REGISTRY"
+            value: "docker-private-arondor-host.packages.nuxeo.com"
+          - key: "CONNECT_PROD_URL"
+            value: "https://connect.nuxeo.com/nuxeo/site"
```

```suggestion
            value: "https://connect.nuxeo.com/nuxeo"
```
If we are talking about a "Connect prod URL"

---

Replaced by `CONNECT_PROD_SITE_URL`.